### PR TITLE
Update UG 

### LINKED ISSUE #190: [Ting Kai] Update UG Command
- Sort all items by an attribute

- Increase the quantity of an item

- Decrease the quantity of an item

### LINKED ISSUE #191: [Richard] Update UG Command
- Create a new item  

- View the information of an item

- Reset the application

### LINKED ISSUE #192: [Bryan] Update UG Command
- Create a new tag

- Rename a tag

- Delete a tag

### PR DESCRIPTION
**Pull request by Eugenetanwl3881**
Update UG in the following ways

- Updated command information for general and tag commands
- Updated command summary table
- Re-ordered certain parts of the UG
- Updated Glossary
- Updated QuickStart
- Update on How to Use User Guide

Refer to https://docs.google.com/document/d/1xQvtcPoOtZJPF-_VDjUxj7taC_0nH-pHcMNJyuiYR1o/edit for full discussion on updates.

Closes #190 	
Closes #191 	
Closes #192

### CODE REVIEW

File: docs/_ug/commandSummary/tagCommands/tag.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 <!-- markdownlint-disable-file first-line-h1 -->
-<b>tag n/TAG_NAME id/INDEX_LIST</b><br><br><u>Valid Examples:</u><br>tag n/Condiments id/1<br>tag n/Condiments id/1,2,4,8<br><br><u>Invalid Example:</u><br>tag Condiments 1
+`tag n/TAG_NAME id/INDEX`
```
Comment (RichDom2185) [inline docs/_ug/commandSummary/tagCommands/tag.md:2]: I thought we are standardising to remove the `id/` prefix entirely?

```suggestion
`tag n/TAG_NAME INDEX`
```

File: docs/_ug/commandSummary/tagCommands/untag.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
 <!-- markdownlint-disable-file first-line-h1 -->
-<b>untag n/TAG_NAME id/INDEX_LIST</b><br><br><u>Valid Examples:</u><br>untag n/Condiments id/1<br>untag n/Condiments id/1,2,4,8<br><br><u>Invalid Example:</u><br>untag Condiments 1,2
+`untag n/TAG_NAME id/INDEX`
```
Comment (RichDom2185) [inline docs/_ug/commandSummary/tagCommands/untag.md:2]: Ditto. Should we change to the following?

```suggestion
`untag n/TAG_NAME INDEX`
```

**Review by RichDom2185 — CHANGES_REQUESTED** [review]
Overall looks good. There are some auto-fixable style issues (which I have left out from this review), but the non-auto-fixable issues have been pointed out below. I will fix most of them. There are also some clarifications I want to ask; the details will be below once I've resolved the other issues. Thanks!

File: docs/_ug/QuickStart.md [hunk 3]
```diff
@@ -1,78 +1,39 @@
...
+If you haven't installed FoodRem, simply follow these steps to set it up:
 
-The format for commands are not identical. One command in FoodRem is the command to create a new item.
+1. Ensure you have [Java 11](https://docs.oracle.com/en/java/javase/11/install/overview-jdk-installation.html) installed on your computer.
+2. Download the latest foodrem.jar file from our [release page](https://github.com/AY2223S1-CS2103T-W16-2/tp/releases/)
+3. Copy the jar file to an empty folder. This will be your home folder for FoodRem.
+4. Double-click on the jar file to launch FoodRem.
 
-Format: `new n/ITEM_NAME [qty/QUANTITY] [unit/UNIT] [bgt/BOUGHT_DATE] [exp/EXPIRY_DATE]`
+ℹ If FoodRem does not launch after double-clicking the jar file, check out the [Troubleshooting](#troubleshooting) section for an alternative solution.
```
[the remark this answers is not in the document]
Comment (Eugenetanwl3881) [reply docs/_ug/QuickStart.md:12]: Symbol for additional info (refer to Quickstart). Perhaps we can change this to an admonition? (If that's the case and we are  not using those symbols in Quickstart at all anymore, we can remove later on)

File: docs/_ug/QuickStart.md [hunk 4]
```diff
@@ -1,78 +1,39 @@
...
 
-Note:
+| Name               | Description                                      |
+|--------------------|--------------------------------------------------|
+| Command Input Box  | ![CommandInputBox](images/CommandInputBox.png)   |
+|
+| Command Output Box | ![CommandOutputBox](images/CommandOutputBox.png) |
+|
+| List Box           | ![FullUiImage](images/ListBox.png)               |
+|
```
[the remark this answers is not in the document]
Comment (Eugenetanwl3881) [reply docs/_ug/QuickStart.md:34]: Ok will fix. Didn't realise

File: docs/_ug/TryingFirstCommand.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+`UNIT`: kg
+
+`BOUGHT_DATE`: 05-09-2022
+
+
+Note:
+
+* The [Placeholder](#placeholders) section covers the restrictions for respective placeholders.
+  For example, the date format of BOUGHT_DATE, certain characters you cannot use and the limit and precision of numbers.
```
[the remark this answers is not in the document]
Comment (Eugenetanwl3881) [reply docs/_ug/TryingFirstCommand.md:27]: Oh I didn't mean to have a second line for the example. Can keep all in one line. Fixed

File: docs/_ug/commandSummary/tagCommands/tag.md (hunk 1 above)
Comment (Eugenetanwl3881) [reply docs/_ug/commandSummary/tagCommands/tag.md:2]: Oh, I realise our tag impelementation is still using id/. We'll put id/ for now until we change up the implementation and testing. Will update this after implementation is updated.

File: docs/_ug/commandSummary/tagCommands/untag.md (hunk 2 above)
Comment (Eugenetanwl3881) [reply docs/_ug/commandSummary/tagCommands/untag.md:2]: Ditto. Will change after implemnetation is changed

File: docs/_ug/Placeholders.md [hunk 6]
```diff
@@ -18,24 +20,27 @@
 | Placeholders | Related Flags | Description   |
 |--------------|---------------|---------------|
 | INDEX        | id/           | :INDEX:       |
-| INDEX_LIST   | id/           | :INDEX_LIST:  |
```
[the remark this answers is not in the document]
Comment (Eugenetanwl3881) [reply docs/_ug/Placeholders.md:21]: Fixed

File: docs/_ug/QuickStart.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Ferusel) [reply docs/_ug/QuickStart.md:12]: Yes, we should make this as an admonition.

**Review by Ferusel — APPROVED** [review]
LGTM!

**Pull request merged**